python_ta/reporters/node_printers.py

```diff
@@ -614,6 +614,36 @@ def render_pep8_errors_e306(msg, _node, source_lines=None):
     yield from render_context(msg.line, msg.line + 2, source_lines)
 
 
+def render_missing_return_statements(msg, _node, source_lines=None):
+    """
+    Render a missing return statements message
+    """
+    line = msg.line
+    end = _node.tolineno
+
+    # render function header and context
+    if line - 1 > _node.fromlineno:
+        yield from render_context(_node.fromlineno, _node.fromlineno + 1, source_lines)
+        if line - 1 > _node.fromlineno + 1:
+            yield (None, slice(None, None), LineType.CONTEXT, '"""MORE CODE OMITTED"""')
+    yield from render_context(line - 1, line + 1, source_lines)
+
+    # calculate indentation for the insertion point
+    body = source_lines[end - 1]
+    indentation = len(body) - len(body.lstrip())
+    insertion_text = body[:indentation] + '"""MISSING RETURN STATEMENT"""'
```

Comment:
Change this to a comment rather than a docstring, and use the more direct instruction `INSERT RETURN STATEMENT HERE`